rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/90_unsupported_operations.yml

```diff
@@ -232,3 +232,91 @@ aggregate on _id:
             id:
               terms:
                 field: _id
+
+---
+synthetic source flattened field:
+  - skip:
+      version: " - 8.6.99"
+      reason: "synthetic source introduced in 8.7.0"
+
+  - do:
+      catch: /field \[k8s.pod.labels\] of type \[flattened\] doesn't support synthetic source/
```

Comment:
we should be fixing that soon :)

Reply:
Is synthetic source the only reason that flattened fields are not supportes in time-series indices?

I think it is hard to support flattened fields as dimensions or metrics. They can only exist as labels, right?

Reply:
I think sub fields of flattened field should be able to be dimensions. This doesn't work now. But I think we can support that too? I don't think we can support subfields of flattened fields to be metrics.